Makefile

```diff
@@ -1,11 +1,11 @@
 GOPATH?=~/go
 GOBIN?=${GOPATH}/bin
 
-PROTOS = $(wildcard networkservice/*.proto)
+PROTOS = $(wildcard pkg/*/apis/**/*.proto)
 PBS = $(PROTOS:%.proto=%.pb)
 
 all: protos
 protos: ${PBS}
 
 %.pb: %.proto
-	PATH=~/bin:$(PATH):$(GOBIN) protoc -I networkservice $*.proto --go_out=plugins=grpc:go
+	PATH=~/bin:$${PATH}:$(GOBIN) protoc -I $(dir $*.proto) $*.proto --go_out=plugins=grpc:$(dir $*.proto)
```

Comment:
Can you double check this? The last flag should probably be --go_out=plugins=grpc:go